bundle/manifests/backstage-default-config_v1_configmap.yaml

```diff
@@ -62,9 +62,13 @@ data:
             janus-idp.io/app: backstage-psql-cr1 # placeholder for 'backstage-psql-<cr-name>'
```

Comment:
For my understanding: Why reviewer should spend time reviewing autogenerated files?

Reply:
To me, even if it is generated, this is also part of the code; just like the manifests generated via the [kubebuilder marker comments](https://book.kubebuilder.io/reference/generating-crd). It also seems quite common in the operators world, no? ([opentelemetry-operator](https://github.com/open-telemetry/opentelemetry-operator/tree/main/bundle), [observability-operator](https://github.com/rhobs/observability-operator/tree/main/bundle), [jaeger-operator](https://github.com/jaegertracing/jaeger-operator/tree/main/bundle), [grafana-operator](https://github.com/grafana/grafana-operator/tree/master/bundle), ...)
It also makes it easier to check them with code-scanning tools like SonarCloud, Snyk.

Reply:
Yes, generated code should also be reviewed. The fact that the code is generated does not mean it works as intended.

Reply:
You can review whatever you want but, honestly, how often you review for example zz_generated.deepcopy.go file?

If you tend not trust the tool the best thing is go and check it once (maybe trying theirs tests etc), otherwise it become nightmare.
Thats how separation of concern works I think.